### PR TITLE
Add proton path detection

### LINKED ISSUE #11: Client nonfunctional with Among Us installed on Linux through Steam
The client does not function when attempting to patch a Steam installation of Among Us on Linux, complaining that the Among Us install directory doesn't exist.

I've tried executing the client with both Proton and straight Wine (from within Steam's wineprefix); it otherwise runs fine.

I added a symlink `drive_c/users/<username>` pointing to `drive_c/users/steamuser`, suspecting that it was looking at the wrong user directory and thus not finding AppData; this did absolutely nothing, surprisingly.

I have not tested in vanilla Wine completely without Steam as I do not own a DRM-free (itch.io) copy of Among Us.

This could easily be solved by simply adding a manual override for `regionInfo.dat`'s location; this would even allow the client to be run natively with mono on Linux instead of running within Wine.

### PR DESCRIPTION
**Pull request by js6pak**
Fixes #11
Migrates client's .csproj to new format allowing building it under mono.

### CODE REVIEW

**Comment by AeonLucid** [comment]
Sorry but I don't want to merge csproj format until .NET 5 is released. 
GitHub CI has issues with .NET 5 in the new csproj format because the msbuild is outdated on the CI server.

**Comment by js6pak** [comment]
Well, I can't build old csproj format soo? This PR can wait I guess.

File: src/Impostor.Client/Impostor.Client.csproj [hunk 1]
```diff
@@ -1,102 +1,28 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (joaosombrio) [inline src/Impostor.Client/Impostor.Client.csproj:1]: I guess having the new csproj is great but why change the target framework though?

Comment (js6pak) [reply src/Impostor.Client/Impostor.Client.csproj:1]: `System.Resources.Extensions` (https://github.com/dotnet/msbuild/issues/4704) requires netstandard2.0 (net461)

**Comment by AeonLucid** [comment]
Merged with #50.

**Pull request closed without merging**